README.md

```diff
@@ -128,6 +129,7 @@ Differences between the formats:
 - Depending on the use case, the custom (un-)marshal methods of one of the formats might be easier to implement
     - JSON: [`MarshalJSON() ([]byte, error)`](https://pkg.go.dev/encoding/json#Marshaler) and [`UnmarshalJSON([]byte) error`](https://pkg.go.dev/encoding/json#Unmarshaler)
     - gob: [`GobEncode() ([]byte, error)`](https://pkg.go.dev/encoding/gob#GobEncoder) and [`GobDecode([]byte) error`](https://pkg.go.dev/encoding/gob#GobDecoder)
+    - proto: [`Marshal(proto.Message) ([]byte, error)`](https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson#Marshal) and [`Unmarshal([]byte, proto.Message) error`](https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson#Unmarshal)
```

Comment:
With the PR addressing proto (un)marshalling and not protojson, are these the right links?

Reply:
good catch, fixed